profiler/src/main/java/com/splunk/opentelemetry/profiler/allocation/exporter/PlainTextAllocationEventExporter.java

```diff
@@ -39,6 +39,9 @@
 import jdk.jfr.consumer.RecordedThread;
 
 public class PlainTextAllocationEventExporter implements AllocationEventExporter {
+
+  private static final java.util.logging.Logger LOGGER =
```

Comment:
nit: in this project we use lowercase `logger`
any reason for not importing `java.util.logging.Logger`?

Reply:
I made the var name lowercase for consistency. Thanks!